scripts/stream.py

```diff
@@ -69,15 +69,19 @@ def main():
     pipe.Add(stream_root.builder)
     pipe.Add(sosmurf.SessionManager.SessionManager, stream_id=args.stream_id)
     # pipe.Add(sosmurf.util.stream_dumper)
-    pipe.Add(core.G3NetworkSender, hostname='*',
-             port=args.stream_port, max_queue_size=1000)
+    # pipe.Add(core.G3NetworkSender, hostname='*',
+    #          port=args.stream_port, max_queue_size=1000)
```

Comment:
Is this supposed to still be commented out?

Reply:
nope